cmake/cmaize/targets/installed_target.cmake

```diff
@@ -0,0 +1,36 @@
+include_guard()
+include(cmakepp_lang/cmakepp_lang)
+
+#[[[
+# Wraps a target that is already installed on the system.
+#]]
+cpp_class(InstalledTarget)
```

Comment:
Isn't this supposed to inherit from Target?

Reply:
Oh, you're right... good catch.